src/box/lua/config/applier/credentials.lua

```diff
@@ -1,4 +1,5 @@
 local log = require('internal.config.utils.log')
```

Comment:
Regarding the second commit and its NO_DOC/NO_CHANGELOG statements.

> NO_CHANGELOG=internal
> NO_DOC=internal

At least, now pap-sha256 is fully supported (on Tarantool EE). A password also can be removed after the changes. These are user-visible changes. I propose to add a changelog entry.

I propose to state that https://github.com/tarantool/doc/issues/3544 remains actual after the changes in the NO_DOC clause.

Reply:
Done:
```md
## feature/config

* Implemented full password support in config.credentials schema, 
  including password set, update, removal for `chap-sha1` auth type
  (supported by both Tarantool Community Edition and Tarantool Enterprise
  Edition) and `pap-sha256` (just for Enterprise Edition where it is
  available) (gh-8967).
```

Reply:
Updated NO_DOC too.